modules/openapi-generator/src/main/resources/typescript-node/api-all.mustache

```diff
@@ -8,5 +8,9 @@ import { {{ classname }} } from './{{ classFilename }}';
 export * from './{{ classFilename }}Interface'
 {{/withInterfaces}}
 {{/apis}}
+import http = require('http');
+export class HttpError {
+    constructor (public response: http.{{#supportsES6}}IncomingMessage{{/supportsES6}}{{^supportsES6}}ClientResponse{{/supportsES6}}, public body: any, public statusCode?: number) { }
```

Comment:
```suggestion
    constructor (public response: http.{{#supportsES6}}IncomingMessage{{/supportsES6}}{{^supportsES6}}ClientResponse{{/supportsES6}}, public body: any, public statusCode?: number) {
      this.name = 'HttpError'
    }
```

Reply:
should I also add a message? Something like `HTTP request failed`?

Reply:
yes, this would be helpful